packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -2794,6 +2834,26 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
   getCallRtpStats(): Promise<CallRtpStats> {
     return this.getCallStats();
   }
+
+  /**
+   * Handle timeout for the missed events
+   * @param expectedStates - An array of next expected states
+   * @param errorMessage - Error message to be emitted if the call is not in the expected state in expected time
+   */
+  private async handleTimeout() {
+    log.warn(`Call timed out`, {
```

Comment:
state machine doesnt store the previous state. So this is all the informatino that we can log

Reply:
But the last state is present in the logs so we can still get the previous state